packages/fxa-shared/test/subscriptions/stripe.ts

```diff
@@ -25,6 +25,10 @@ const subscriptionExpanded = require('../fixtures/stripe/subscription_expanded.j
 const price1 = require('../fixtures/stripe/price1_with_product.json');
 const plan = require('../fixtures/stripe/plan1_with_product.json');
 
+function deepCopy(object: any) {
+  return JSON.parse(JSON.stringify(object));
+}
+
```

Comment:
I thought we had this in an easier to use spot now to avoid having to copy it around. A `util` module around here?

Reply:
So not that I found but I also actually ended up not needing this and just used the `...` operator to be consistent with how the other tests were written in this file